runtime/doc/eval.txt

```diff
@@ -4891,7 +4892,7 @@ readfile({fname} [, {binary} [, {max}]])
 		separated with CR will result in a single long line (unless a
 		NL appears somewhere).
 		All NUL characters are replaced with a NL character.
-		When {binary} is equal to "b" binary mode is used:
+		When {binary/append} contains "b" binary mode is used:
```

Comment:
This one confused me, because readfile doesn't even take an append option. The code also doesn't do anything different as far as I can tell when the append option is given.